Cakefile

```diff
@@ -166,6 +169,30 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
   console.log "built ... running browser tests:"
   invoke 'test:browser'
 
+task 'build:inline_runtime', 'build the inline iced3 runtime', ->
+  runtime_dir = path.dirname require.resolve 'iced-runtime-3'
+  code = ''
+  for name in ['const', 'runtime', 'library', 'main']
+    code += """
+      require['./#{name}'] = (function() {
+        var exports = {}, module = {exports: exports};
+        #{fs.readFileSync "#{runtime_dir}/#{name}.js"}
+        return module.exports;
+      })();
+    """
+  code = """
+    (function() {
+      function require(path){ return require[path]; }
+      #{code}
+      return require['./main'];
+    }());
+  """
+
+  unless process.env.MINIFY is 'false'
+    {code} = require('uglify-js').minify code, fromString: true
+
```

Comment:
Talking to @chrisnojima about this in the office, we probably shouldn't be outputting ES6 to browsers anyways. People should convert to ES6 via babel (or whatnot).  This is of course a much deeper concern, and indicates a deeper change in workflow, so maybe we shouldn't spend too many cycles on this.